src/main/java/net/dv8tion/jda/api/entities/MessageReaction.java

```diff
@@ -221,15 +220,15 @@ public GuildMessageChannel getGuildChannel()
     }
 
     /**
-     * The {@link net.dv8tion.jda.api.entities.MessageReaction.ReactionEmote ReactionEmote}
-     * of this Reaction
+     * The {@link Emoji} of this Reaction.
+     * <br>This includes both {@link Emoji.Type#CUSTOM custom emojis} and {@link Emoji.Type#UNICODE unicode emoji}.
```

Comment:
should these maybe refer to the classes instead for cast and method discovery?

Reply:
For this case, referring to type is more useful since you are getting not creating